app/views/hiring_staff/vacancies/job_specification/edit.html.haml

```diff
@@ -31,6 +31,25 @@
                 wrapper_html: {id: 'subject'},
                 collection: Subject.order(:name),
                 required: false
+      %fieldset.form-group#pay_scale_range
+        %label.form-label.form-label-bold= t('vacancies.pay_scale_range')
+        %span.form-hint= t('vacancies.form_hints.pay_scale_range')
+
+        = f.input :min_pay_scale_id,
+                  collection: pay_scale_options,
+                  label_method: :label,
+                  label: false,
+                  required: false,
+                  wrapper: false
+
+        to
+
+        = f.input :max_pay_scale_id,
+                  collection: pay_scale_options.reverse,
```

Comment:
When you click this drop down you get the following content which fills the entire height of my screen:

![screen shot 2018-05-02 at 12 07 39](https://user-images.githubusercontent.com/912473/39520147-7a4efdbe-4e01-11e8-8fb7-d919d1585abf.png)


This AC isn't totally clear in regards to the second field but it is the only field referencing ordering and doesn't talk about reversing:
> - order content of drop down as main, then upper, then lead (ascending ££ value and from most to least commonly used)

Reply:
From the trello card:
`both dropdowns are optional ([blank] to [UPS 5], [MPS3] to [blank]`

I had asked @Ellayanor about this as at first I thought the order might need to be different since the leadership scales were not mentioned, who confirmed that this meant reverse order.

Regarding the height, I don't get this behavior on my local so maybe it has to do with your local browser settings. But, it's also a QA issue
<img width="535" alt="selected_max_payscale" src="https://user-images.githubusercontent.com/159200/39521936-85adcef4-4e08-11e8-9cdc-3b0e380f9e9d.png">

Reply:
hey team - I think this one is on me - I tried to give an example to make it clearer but think I made it worse, sorry. 

There are two aspects: 
1. order content of drop down as main, then upper, then lead (ascending ££ value and from most to least commonly used). The order should be the same for both dropdowns. 

2. it should be accepted if a user **only** makes a selection from either one of the dropdowns, leaving the other one blank.